dcrpool.go

```diff
@@ -250,9 +250,15 @@ func main() {
 		}
 	}()
 
-	p, err := newPool(cfg)
+	db, err := pool.InitBoltDB(cfg.DBFile)
+	if err != nil {
+		mpLog.Errorf("Failed to initialize database: %v", err)
```

Comment:
let's not capitalize the errors, to be consistent with the rest of the error definitions.

Reply:
Good catch. Fixed these and a couple of extra ones